tests/filecheck/mlir-conversion/with-mlir/dialects/transform/transform_types.mlir

```diff
@@ -1,6 +1,10 @@
 // RUN: xdsl-opt %s | xdsl-opt | mlir-opt --allow-unregistered-dialect | filecheck %s
```

Comment:
why is `--allow-unregistered-dialect` necessary here?

Reply:
all other tests used it, so I used it too... but indeed it is not strictly necessary here

Reply:
Let's remove it then!